app/index.html

```diff
@@ -1,6 +1,6 @@
 ---
 id: homepage
-title_override: Kong - Open-Source Microservice and API Management Layer
+title_override: Kong - The Open-Source API Gateway and Microservice Management Layer
```

Comment:
The `The` actually takes too much space in the limited space offered in the tab title imo, I'd rather have people see "Open-source API Mgmt..." instead of "The open-source API..."